components/PlantCard/index.js

```diff
@@ -13,17 +13,19 @@ export default function PlantCard({
 }) {
   return (
     <StyledCard>
+      <Link href={`/plants/${plantId}`}>
       <StyledImageContainer>
         <PlantOwnedButton
           plantId={plantId}
-          onClick={() => handleToggleOwned(plantId)}
+          onClick={(event) => {event.preventDefault(); handleToggleOwned(plantId)}}
```

Comment:
The handleToggleOwned function is passed directly into the button as an inline callback. While this works, it is more efficient and readable to define the function beforehand, especially as the project grows.

My suggestion:

function handleButtonClick(event) {
  event.preventDefault();
  handleToggleOwned(plantId);
}

<PlantOwnedButton
  plantId={plantId}
  onClick={handleButtonClick}
  isOwned={isOwned}
/>

Reply:
Hi @predrag-babic , thanks a lot for this suggestion. Makes perfect sense to us and we decided to change our code accordingly 👍🏻